main/integration-tests/__test__/api-tests/studies/list-studies.test.js

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
What do you think of renaming the file to  get-studies.test.js?  this will make it consistent with get-study.test.js  and matches the fact that list is just 'get' on the collection resource?